src/views/blocks/attendees/description.php

```diff
@@ -8,28 +8,36 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link https://m.tri.be/1amp Help article for RSVP & Ticket template files.
+ * @link    https://m.tri.be/1amp Help article for RSVP & Ticket template files.
  *
- * @since 4.9.2
- * @version 4.11.3
+ * @since   4.9.2
+ * @since   4.11.3
  *
+ * @version TBD Updated docblock vars and removed unnecessary second retrieving of Post ID.
+ *
+ * @var Tribe__Tickets__Editor__Template $this       Template object.
+ * @var int                              $post_id    [Global] The current Post ID to which tickets are attached.
+ * @var array                            $attributes [Global] Attendee block's attributes (such as Title above block).
+ * @var array                            $attendees  [Global] List of attendees with attendee data.
+ * @var array                            $attendees  List of attendees and their data.
  */
+
 $display_subtitle = $this->attr( 'displaySubtitle' );
 
 if ( is_bool( $display_subtitle ) && ! $display_subtitle ) {
 	return;
 }
 
-$post_id         = $this->get( 'post_id' );
 $attendees_total = count( $attendees );
-$message         = _n( 'One person is attending %2$s', '%d people are attending %s', $attendees_total, 'event-tickets' );
+
+$message = _n( 'One person is attending %2$s', '%d people are attending %s', $attendees_total, 'event-tickets' );
 ?>
-<p><?php
-	echo esc_html(
+<p>
+	<?php echo esc_html(
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: fcc422af5b62cf0da8fcb85fae85c135</sub>

Reply:
```suggestion
	<?php
	echo esc_html(
```